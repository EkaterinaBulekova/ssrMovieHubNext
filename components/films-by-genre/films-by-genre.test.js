import React from 'react';
import { shallow } from 'enzyme';
import FilmsByGenre from './films-by-genre';
import toJson from 'enzyme-to-json';


describe('<FilmsByGenre />', () => {
  it('should render button with right name, class and onClick method', () => {
    const testGenre = "Test genre";
    const testClass = "films-by-genre";
    const textTestClass = "films-by-genre-text";
    const component = shallow(<FilmsByGenre/>);
    expect(toJson(component)).toMatchSnapshot();
    // expect(component.hasClass(testClass)).toBeTruthy();
    // expect(component.find('.'+textTestClass).text()).toBe("Films by " + testGenre + " genre");
  });
});