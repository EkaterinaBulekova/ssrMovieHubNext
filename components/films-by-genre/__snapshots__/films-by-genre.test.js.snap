// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FilmsByGenre /> should render button with right name, class and onClick method 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
